minetest/minetest
# Always build locale files in CMAKE_SOURCE_DIR if building RUN_IN_PLACE

Fixes #12038.

## To do

This PR is a Ready for Review.

## How to test

* Remove the `locale` dir in minetest.
* Make an out-of-tree build: `$ cmake <path to minetest> -DRUN_IN_PLACE=1`
* Run minetest (from a folder that is not your build folder): `$ <path to minetest>/bin`
* Notice how translations in main-menu works. In master you get a warning: `Couldn't find a locale directory!`
